CHANGELOG

```diff
@@ -380,7 +380,9 @@ Miscellaneous
 
 - Upgraded the SUNDIALS interface to utilize SUNDIALS 5.0. This necessitated a
   complete rework of the interface and requires changes at the application
-  level. Example usage of the new interface can be found in examples/sundials.
+  level. Basic support for using the CVODES has been added and requires using
+  the cvodes interface instead of the cvode. Example usage of the new interface
+  can be found in examples/sundials.
```

Comment:
This does not belong here -- please revert the changes to the 4.1 section and add a new paragraph in the 4.1.1 section. A good description should explain what new feature is available to the user (solving adjoint problems), what is required for it (certain build of SUNDIALS) and where to see it in action (miniapps/adjoint).

Reply:
Working on this now.